configs/samples/pjsip.conf.sample

```diff
@@ -432,6 +432,16 @@
 ;type=aor
 ;max_contacts=2
 
+;===============DEFAULT ENDPOINT FOR OUTBOUND REQUESTS TO URI===================
+;
+; This is an example default outbound endpoint, which is required when sending
+; an outbound request to a URI without a specified endpoint. The context should
+; be set appropriately for the outbound request.
```

Comment:
"context" isn't used at all for outbound requests, only inbound.
You should also state that the global "default_outbound_endpoint" parameter needs to be set to this.